servicecontrol/servicecontrol-in-practice.md

```diff
@@ -1,35 +1,40 @@
 ---
 title: ServiceControl in Practice
-summary: Details how to think about the use of ServiceControl in production environments
+summary: Considerations for configuring ServiceControl for use in production environments
```

Comment:
"Optimizing ServiceControl for use in Production" sounds better to me. 
Although you are technically configuring SC, you're still providing optimization advice.